lib/src/key/table/lq.rs

```diff
@@ -19,7 +19,6 @@ pub struct Lq<'a> {
 	_d: u8,
 	_e: u8,
 	_f: u8,
-	#[serde(with = "uuid::serde::compact")]
```

Comment:
Hey @rushmorem the UUIDs i the `key` table are internal only. This ensures that the UUIDs are compact in the key entires. What's the reason for removing these lines?

Reply:
https://github.com/surrealdb/surrealdb/issues/2514

Reply:
Thanks @tobiemh! I wasn't aware of that. I thought those needed to be removed as well. I have reverted those changes.

@phughk This tests are still failing on [this assert](https://github.com/surrealdb/surrealdb/blob/99e84d8f775be373e42926b5a34adfca2217592f/lib/src/key/thing/mod.rs#L87) though. If I remove it, like how it was before https://github.com/surrealdb/surrealdb/pull/2166, everything parses. I thought you might have a proper fix for that though.

Reply:
Right I see. I think that was the versioning, actually. I thought it was about the nesting of uuids in keys. If it works as intended now, then great :)
Thanks both!